README.md

```diff
@@ -213,7 +185,7 @@ python manage.py createsuperuser
 
 <details>
 <summary>
-Step 14: Running the Project in local server
+Step 12: Running the Project in local server
```

Comment:
Hi @Saup21, In `Step 12: Running the Project in local server`, Can you please fix the localhost URL in the code block?
Instead of URL in the code block, coming out of the code block, we can write something as 'Server will be up and running on PORT 8000'

Reply:
yeah sure, but should i close this PR and make a new one??

Reply:
Hi, I'll close this PR since it's got conflicts.
Please make a new PR.

Reply:
sure

Reply:
Update your forked repository since `readme.md` is changed now.